map_function.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="./movies.js"></script>
+    <script type="text/javascript">
+        
+        function mapit (item) {
+            var name = [item.title];
```

Comment:
You're putting the title inside an array here. This will create an array of arrays. `var name = item.title` is what you want.

Reply:
ohh understood, thankds